main/editor.js

```diff
@@ -19,11 +19,27 @@ let editor = null;
 
 const openEditorWindow = () => {
   if (!editor) {
+    const width = 768;
+    const barHeight = 48;
+    const height = 768 * 9 / 16 + barHeight;
     editor = new BrowserWindow({
-      width: 400,
-      height: 200
+      minWidth: width,
+      minHeight: height,
+      webPreferences: {
+        webSecurity: false
```

Comment:
Does anyone know if we can load the local `.mp4` without needing to disable web security? 🤔

Reply:
No, it looks like it's not there: https://github.com/wulkano/kap/blob/30ff93afcf2f0b467547a8b70b3f7f0d03ffcbeb/app/src/main/index.js#L587

Reply:
And it doesn't work now without it? It might have something to do with Electron 2.0. I think they are trying to make it more secure

Reply:
Looks like it... 2.0 is using the absolute path without `file://`:

Reply:
I opened an issue over at electron: https://github.com/electron/electron/issues/12942